### PR TITLE
Feature/renaming api 3

### PR DESCRIPTION
**Pull request by lamaralmp**
## Description

Renomeamos el producto API para:

EN | Checkout API
ES | Checkout API
PT | Checkout Transparente

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/payments/api/other-payment-ways.en.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Integrate other payment methods
 
-With Mercado Pago payment API, you can add **alternative payment methods for your customers to make their payments**.
+----[mlb]----
+With Transparent Checkout of Mercado Pago, you can add **alternative payment methods for your customers to make their payments**.
```
Comment (Juanii95) [inline guides/payments/api/other-payment-ways.en.md:4]:
```suggestion
With Checkout API of Mercado Pago, you can add **alternative payment methods for your customers to make their payments**.
```

Comment (Juanii95) [reply guides/payments/api/other-payment-ways.en.md:4]: @pamalv7

File: guides/payments/api/previous-requirements.en.md [hunk 2]
```diff
@@ -106,7 +106,12 @@ Once your integration is ready, you just need to fill out the form for to go to
 >
 > Create and configure your own payment experience.
 >
-> [Integrate Mercado Pago API for cards](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/payments/api/receiving-payment-by-card/)
+----[mlb]----
+> [Integrate Transparent Checkout for cards](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/payments/api/receiving-payment-by-card/)
```
Comment (Juanii95) [inline guides/payments/api/previous-requirements.en.md:110]: @pamalv7

File: guides/payments/api/introduction.en.md [hunk 3]
```diff
@@ -1,4 +1,9 @@
-# Mercado Pago APIs
+----[mlb]----
+# Checkout Transparente
```
Comment (Juanii95) [inline guides/payments/api/introduction.en.md:2]: Va` Checkout API` en ingles

File: guides/payments/api/previous-requirements.en.md (hunk 2 above)
Comment (Juanii95) [inline guides/payments/api/previous-requirements.en.md:110]:
```suggestion
> [Integrate Checkout API for cards](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/payments/api/receiving-payment-by-card/)
```

File: guides/payments/api/receiving-payment-by-card.en.md [hunk 4]
```diff
@@ -1,14 +1,29 @@
-# Integrate the payment API for cards
+----[mlb]----
+# Integrate the Transparent Checkout API payment for cards
```
Comment (Juanii95) [inline guides/payments/api/receiving-payment-by-card.en.md:2]:
```suggestion
# Integrate the Checkout API payment for cards
```

File: guides/payments/api/receiving-payment-by-card.en.md [hunk 5]
```diff
@@ -1,14 +1,29 @@
...
+------------
+----[mla, mlm, mpe, mco, mlu, mlc]----
+# Integrate the Checkout API payment for cards
+------------
 
-Integrating Mercado Pago's payment API for cards allows you to offer a complete payment option within the site. The entire experience takes place in your store, so customers avoid exiting your site while making a purchase.
+----[mlb]----
+Integrating Mercado Pago's Transparent Checkout for cards allows you to offer a complete payment option within the site. The entire experience takes place in your store, so customers avoid exiting your site while making a purchase.
```
Comment (Juanii95) [inline guides/payments/api/receiving-payment-by-card.en.md:9]: @pamalv7 no me suena esto

File: guides/payments/api/receiving-payment-by-card.en.md [hunk 6]
```diff
@@ -1,14 +1,29 @@
...
 
 ## How does it work?
 
 ![API-integration-flowchart](/images/api/api-integration-flowchart-en.png)
 
 <br>
 
-With our Mercado Pago payment API, you should take into account two aspects: data capture and payment confirmation submission.
+----[mlb]----
+With our Mercado Pago Transparent Checkout, you should take into account two aspects: data capture and payment confirmation submission.
```
Comment (Juanii95) [inline guides/payments/api/receiving-payment-by-card.en.md:22]:
```suggestion
With our Mercado Pago Checkout API, you should take into account two aspects: data capture and payment confirmation submission.
```

File: guides/payments/api/receiving-payment-by-card.en.md (hunk 5 above)
Comment (Juanii95) [inline guides/payments/api/receiving-payment-by-card.en.md:9]:
```suggestion
Integrating Mercado Pago's Checkout API for cards allows you to offer a complete payment option within the site. The entire experience takes place in your store, so customers avoid exiting your site while making a purchase.
```

File: guides/payments/api/previous-requirements.en.md (hunk 2 above)
Comment (pamalv7) [reply guides/payments/api/previous-requirements.en.md:110]: En inglés siempre **Checkout API**

File: guides/payments/api/receiving-payment-by-card.en.md (hunk 5 above)
Comment (pamalv7) [reply guides/payments/api/receiving-payment-by-card.en.md:9]: Dejar única versión

File: guides/payments/api/receiving-payment-by-card.es.md [hunk 7]
```diff
@@ -1,14 +1,29 @@
...
+----[mlb]----
+# Integra el Checkout Transparente para pagos con tarjetas
+------------
+----[mla, mlm, mpe, mco, mlu, mlc]----
+# Integra el Checkout API para pagos con tarjetas
+------------
 
-La integración por API de pagos de Mercado Pago para tarjetas permite que puedas ofrecer una opción de pagos completa dentro de tu sitio. Toda la experiencia sucede en tu tienda para que los clientes no tengan que salir al momento de realizar la compra.
+----[mlb]----
+La integración por Checkout Transparente de Mercado Pago para tarjetas permite que puedas ofrecer una opción de pagos completa dentro de tu sitio. Toda la experiencia sucede en tu tienda para que los clientes no tengan que salir al momento de realizar la compra.
```
Comment (pamalv7) [inline guides/payments/api/receiving-payment-by-card.es.md:9]:
```suggestion
La integración del Checkout Transparente de Mercado Pago para tarjetas permite que puedas ofrecer una opción de pagos completa dentro de tu sitio. Toda la experiencia sucede en tu tienda para que los clientes no tengan que salir al momento de realizar la compra.
```

File: guides/payments/api/receiving-payment-by-card.es.md [hunk 8]
```diff
@@ -1,14 +1,29 @@
...
+------------
+----[mla, mlm, mpe, mco, mlu, mlc]----
+La integración por Checkout API de Mercado Pago para tarjetas permite que puedas ofrecer una opción de pagos completa dentro de tu sitio. Toda la experiencia sucede en tu tienda para que los clientes no tengan que salir al momento de realizar la compra.
```
Comment (pamalv7) [inline guides/payments/api/receiving-payment-by-card.es.md:12]:
```suggestion
La integración del Checkout API de Mercado Pago para tarjetas permite que puedas ofrecer una opción de pagos completa dentro de tu sitio. Toda la experiencia sucede en tu tienda para que los clientes no tengan que salir al momento de realizar la compra.
```

**Pull request merged**